clevertap-pushtemplates/src/main/java/com/clevertap/android/pushtemplates/content/PendingIntentFactory.kt

```diff
@@ -85,7 +90,7 @@ internal object PendingIntentFactory {
         intent.putExtra(PTConstants.PT_DISMISS_INTENT, true)
 
         var flagsLaunchPendingIntent = PendingIntent.FLAG_CANCEL_CURRENT
-        if (Build.VERSION.SDK_INT >= VERSION_CODES.S) {
+        if (VERSION.SDK_INT >= VERSION_CODES.M) {
```

Comment:
Is this intentional?

Reply:
Yess. There is no reason to not set the flag for lower api version. Whoever added it, missed it I think
I have tested the changes as well